### PR TITLE
Various Map Icon Fixes

### PR DESCRIPTION
**Pull request by geodavey**
Some map icons display at a smaller size than they should (#494). Pretty sure this is due to their native resolution being 16x16 when it should be 24x24.

```
> grep -l "0 0 16 16" data/styles/clear/style-clear/*-m.svg
car-repair-m.svg
charging-station-m.svg
museum-m.svg
```

This pull request:
1. Updates hostel-m icon to a filled svg (#530)
2. Resizes the above icons, in style-clear and style-night, to 24x24 (opened them up in inkscape)
3. Replaces symbols with `./tools/unix/generate_symbols.sh `

Symbols before: (resources-6plus_dark)
symbols-before

Symbols after: (resources-6plus_dark)
symbols-after

### CODE REVIEW

**Comment by biodranik** [comment]
Did you try to use rsvg-convert to resize svgs? Is it's output better or worse?

**Comment by geodavey** [comment]
@biodranik No I didn't give it a shot as in the end there were just a few icons to handle. Inkscape did add some cruft to the svg tags but it didn't seem to make the difference when running generate-styles. Also I just tried to do a force commit to make the DCO bot happy and looks like it added some commits that were made in the interim. I'll see about fixing that...

File: data/resources-6plus_clear/symbols-ad.sdf [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE skin>
 <root>
  <file width="8" height="8">
-  <symbol minX="0" maxX="7" minY="0" maxY="7" name="00000_keep"/>
+  <symbol minY="0" maxX="7" maxY="7" minX="0" name="00000_keep"/>
```
[the remark this answers is not in the document]
Comment (geodavey) [reply data/resources-6plus_clear/symbols-ad.sdf:4]: Agreed, I saw the same. Looks like a lot of the icons from the resulting sprite PNGs were shifted around

**Comment by geodavey** [comment]
@biodranik apologies for the strange commits I'm a bit new at this, though I think it should be good at this point? Happy to make any adjustments

**Comment by biodranik** [comment]
Thanks! I've cleaned up a bit the pull request and moved your commits to #785

**Pull request closed without merging**